lib/src/async_cache.dart

```diff
@@ -36,6 +38,12 @@ class AsyncCache<T> {
   /// Cached results of a previous [fetch] call.
   Future<T>? _cachedValueFuture;
 
+  ///Default is set to true
+  ///If we set this variable to false
+  ///On the initial run, if callback returned the [Exception]
```

Comment:
It's not necessarily an `Exception`, it's the same if the future could completes with any error.

Reply:
```
  /// Default is set to true
  /// If this variable is set to false,
  /// and callback gets completed with an error,
  /// then the response will not get cached.
```

How about this description?

Reply:
Updated description with the suggested one.